imap.adoc

```diff
@@ -48,3 +48,12 @@ Do NOT configure a load balancer between the Zimbra HTTP Proxy node and any remo
 [IMPORTANT]
 The zmlocalconfig setting *nio_imap_enabled* MUST have value *true* when using the Zimbra IMAPD server.  The legacy
  Imap server which does not use NIO is not supported.
+
+==  IMAPD Configuration settings
+
+The following global and server level configuration attributes are available to control and tune the IMAPD service.
+
+* *zimbraRemoteImapServerEnabled*. When set to TRUE, the standalone IMAP server is enabled. When set to FALSE, the standalone IMAP server is disabled. Default value is FALSE.
```

Comment:
nit: I prefer the use of em dash (--) over period (.) when separating the config key and the description (see https://github.com/Zimbra/adminguide/blob/master/backuprestore.adoc#backup-options for an example). It doesn't appear that we're always consistent with this however (perhaps something we could grep through the adoc's for and cleanup some time).

Also, the four (4) back to back *IMPORTANT* sections really look odd, but not sure how we'd like to fix that (or if we even want to... perhaps put that all in a table?)